<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventswindow</class>
 <widget class="QDialog" name="eventswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="newAttrVal_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>new attr value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addEvent_Butt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>add event</string>
   </property>
  </widget>
  <widget class="QLabel" name="newDevID_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>device</string>
   </property>
  </widget>
  <widget class="QLabel" name="cluster_lbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>cluster</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cluster_comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="attr_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>attribute</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectScen_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select scenario:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="device_comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="attr_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="selectScen_cBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="seconds_spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="hours_spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="newTime_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>time (HH:MM:SS)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minutes_spinBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="attrValue_spinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="maxVal_lbl1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max value: </string>
   </property>
  </widget>
  <widget class="QLabel" name="maxVal_lbl2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="devType_lbl2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="devType_lbl1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Device type:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
